hydra-cluster/src/Hydra/Cluster/Faucet.hs

```diff
@@ -1,6 +1,6 @@
 module Hydra.Cluster.Faucet where
 
-import Hydra.Cardano.Api
+import Hydra.Cardano.Api hiding (cardanoEra, queryCurrentEra)
```

Comment:
Can we switch, in general, to using explicit imports from hydra-cardano-api. `hiding` is in general confusing for provenance and explicit imports while verbose are immediately clear to the reader.